### PR TITLE
POL-135-Updated Permission block

### PR DESCRIPTION
**Pull request by chandravurubindi**
POL-135-Updated Permission block

### Description

Unable to upload Azure Hybrid Use Benefit for Windows Server

### Issues Resolved

Removed comma in the permission block

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt [hunk 1]
```diff
@@ -2,18 +2,14 @@ name "Azure Hybrid Use Benefit for Windows Server"
...
 severity "low"
 category "Cost"
 
-###############################################################################
-# Permissions
-###############################################################################
-
 permission "perm_instances" do
   label "Permissions for instances"
   actions "rs_cm.index", "rs_cm.show"
```
Comment (srpomeroy) [inline cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt]: Not sure why we need this, we aren't using rs_cm.instance resources.
However, we should have the credentials permissions block.

> permission "perm_read_creds" do
> actions   "rs_cm.show_sensitive","rs_cm.index_sensitive"
> resources "rs_cm.credentials"
> end

File: cost/azure/hybrid_use_benefit/README.md [hunk 2]
```diff
@@ -18,17 +18,17 @@ This Policy Template is used to automatically apply the Azure Hybrid Use Benefit
```
Comment (srpomeroy) [inline cost/azure/hybrid_use_benefit/README.md]: Should add `Microsoft.Compute/virtualMachines/write` here as well.

File: cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt [hunk 3]
```diff
@@ -2,18 +2,14 @@ name "Azure Hybrid Use Benefit for Windows Server"
 rs_pt_ver 20180301
 type "policy"
 short_description "Identifies eligible instances not utilizing Azure Hybrid Use Benefit.  See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/azure/hybrid_use_benefit) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```
Comment (cdwilhelm) [inline cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt]: I suggest changing this to **Identifies instances eligible for Azure Hybrid Use Benefit.**

File: cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt (hunk 1 above)
Comment (chandravurubindi) [reply cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt]: Updated the permissions block.

File: cost/azure/hybrid_use_benefit/README.md (hunk 2 above)
Comment (chandravurubindi) [reply cost/azure/hybrid_use_benefit/README.md]: Added Microsoft.Compute/virtualMachines/write.

File: cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt (hunk 3 above)
Comment (chandravurubindi) [reply cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt]: Updated this

File: cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt [hunk 4]
```diff
@@ -1,19 +1,14 @@
...
-# Permissions
-###############################################################################
-
-permission "perm_instances" do
-  label "Permissions for instances"
-  actions "rs_cm.index", "rs_cm.show"
-  resources "rs_cm.instances",
+permission "perm_read_creds" do
+actions "rs_cm.show_sensitive","rs_cm.index_sensitive"
+resources "rs_cm.credentials"
```
Comment (srpomeroy) [inline cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt]: Needs to be properly indented.

Comment (chandravurubindi) [reply cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt]: Sorry, Updated indented properly now.

**Pull request merged**